charts/kubescape-cloud-operator/assets/host-scanner-definition.yaml

```diff
@@ -45,6 +48,13 @@ spec:
           privileged: true
           readOnlyRootFilesystem: true
           procMount: Unmasked
+        env:
+        {{- if .Values.otelCollector.enabled }}
+        - name: ACCOUNT_ID
```

Comment:
[TODO]: Reconsider adding the account ID because we expect Kubescape to add the accountID to the HostScanner environment variables